appmap/_implementation/event.py

```diff
@@ -45,6 +45,15 @@ def reset(cls):
         cls._next_thread_id = 0
 
 
+def describe_value(val):
+    """
+    Return a description of the given value, with the form
+    (class_name, object_id).
+    """
+    return (fqname(type(val)),
+            id(val))
+
+
```

Comment:
How about instead returning a `{class, value, object_id}` dict and then use that to DRY up `Param` and `FuncReturnEvent`?

Reply:
Will do.